static/usage/v8/theming/always-dark-mode/angular/styles_css.md

```diff
@@ -0,0 +1,39 @@
+```css
+/*
+ * App Global CSS
+ * ----------------------------------------------------------------------------
+ * Put style rules here that you want to apply globally. These styles are for
+ * the entire app and not just one component. Additionally, this file can be
+ * used as an entry point to import other CSS/Sass files to be included in the
+ * output CSS.
+ * For more information on global stylesheets, visit the documentation:
+ * https://ionicframework.com/docs/layout/global-stylesheets
+ */
+
+/* Core CSS required for Ionic components to work properly */
+@import '~@ionic/angular/css/core.css';
+
+/* Basic CSS for apps built with Ionic */
+@import '~@ionic/angular/css/normalize.css';
+@import '~@ionic/angular/css/structure.css';
+@import '~@ionic/angular/css/typography.css';
+
+/* Optional CSS utils that can be commented out */
+@import '~@ionic/angular/css/padding.css';
+@import '~@ionic/angular/css/float-elements.css';
+@import '~@ionic/angular/css/text-alignment.css';
+@import '~@ionic/angular/css/text-transformation.css';
+@import '~@ionic/angular/css/flex-utils.css';
+@import '~@ionic/angular/css/display.css';
+
+/**
+ * Ionic Dark Theme
+ * -----------------------------------------------------
+ * For more information, please see:
+ * https://ionicframework.com/docs/theming/dark-mode
+ */
+
+@import '~@ionic/angular/css/themes/dark.always.css';
+/* @import "~@ionic/angular/css/themes/dark.class.css"; */
+/* @import "~@ionic/angular/css/themes/dark.system.css"; */
```

Comment:
I brought this up [in Slack](https://outsystems.slack.com/archives/C06C5HAHURY/p1705078436774389) last week and everyone voted to keep the files commented out. I think the reason we want to keep all of them is because it shows that you need to comment out the file for the dark theme you don't want when changing from the default in the [starters](https://github.com/ionic-team/starters/blob/51dd4f8dca6ae28a1d9db378d168d4b13de8bea2/angular/base/src/global.scss#L28-L37).